libs/features/pages/content-development/src/lib/components/content-dev-hero-section/content-dev-hero-section.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component } from '@angular/core';
+
+import { __heroTexts } from '@elewa-website/data/sections';
+import { SliderButtonData } from '@elewa-website/models/schema/ui/buttons';
+
+@Component({
+  selector: 'elewa-website-content-dev-hero-section',
+  templateUrl: './content-dev-hero-section.component.html',
+  styleUrls: ['./content-dev-hero-section.component.scss'],
+})
+export class ContentDevHeroSectionComponent {
+  heroTexts = __heroTexts
+
+  buttonData: SliderButtonData = {
+    text: "unlock your potential"
+  }
```

Comment:
Nicely done, just one more change. You can also move the buttonData to the same file as that of the 'heroTexts' variable and reference it the same way as that of the 'heroTexts' variable.